ksp_plugin/vessel_subsets.hpp

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <list>
+
+#include "base/disjoint_sets.hpp"
+
+#include "ksp_plugin/pile_up.hpp"
+#include "ksp_plugin/vessel.hpp"
+
+// This ksp_plugin file is in namespace |base| to specialize templates declared
+// therein.
+
+namespace principia {
+namespace base {
+
+template<>
+class Subset<ksp_plugin::Vessel>::Properties {
```

Comment:
It would be good to have an overview of what this specialization is trying to do.  I can read the detailed comments about each method, but I have no idea what the general mechanism is.